Xwt/Xwt/Key.cs

```diff
@@ -27,6 +27,7 @@
 
 namespace Xwt
 {
+	[Flags]
```

Comment:
If I do `a | b`, it'll result in a totally other key, not combine the two keys. I'm possibly missing the point here, but having `Flags` won't make keys combinable.